packages/devtools/lib/src/memory/memory_plotly.dart

```diff
@@ -105,8 +216,8 @@ class MemoryPlotly {
 
     if (marker == null) {
       return Data(
-        x: [],
-        y: [],
+        x: [Null], // Trace legend entry appears w/o data.
```

Comment:
Nit: Trace legend entry appears w/o data --> Null is needed so the trace legend entry appears w/o data.

Reply:
Done.